src/main/java/com/epam/ta/reportportal/ws/model/integration/auth/AbstractAuthResource.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.ta.reportportal.ws.model.integration.auth;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.io.Serializable;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class AbstractAuthResource implements Serializable {
+
+	private String type;
```

Comment:
should it be in integration params? do we want to have a separate resource for auth integrations?

Reply:
done